src/EventTickets/resources/templates/EventTickets/types.ts

```diff
@@ -0,0 +1,17 @@
+export type Event = {
+    id: number;
+    title: string;
+    date: string; // ISO 8601 format date string
```

Comment:
Instead of assuming a specific string format could we cast this to a date object?

Reply:
Refactored here: 949cea38a52d5ede1a19fecc61ef8018c4bb0178